### PR TITLE
Switch to head instead of get when calling exist? method

### PR DESCRIPTION
**Pull request by Lloyd66**
Hi,
This pull request may improve performance as the file are not downloaded anymore when checking their existence.

### CODE REVIEW

File: lib/active_storage/service/open_stack_service.rb [hunk 1]
```diff
@@ -143,6 +143,10 @@ def update_metadata(key, content_type:, disposition: nil, filename: nil, **)
 
   private
 
+    def head_for(key, &block)
+      client.head_object(container, key, &block)
```
Comment (chaadow) [inline lib/active_storage/service/open_stack_service.rb]: [`#head_object`](https://github.com/fog/fog-openstack/blob/master/lib/fog/openstack/storage/requests/head_object.rb)  does not accept a block.

```ruby
 def head_for(key)
      client.head_object(container, key)
end
```

File: lib/active_storage/openstack/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module ActiveStorage
   module Openstack
-    VERSION = '1.4.1'
+    VERSION = '1.4.2'
```
Comment (chaadow) [inline lib/active_storage/openstack/version.rb]: can you remove this file from this commit, i'll update the version myself

**Review by chaadow — COMMENTED** [review]
Thanks for the change!

**Comment by Lloyd66** [comment]
Hi, I amended the commit :)

**Comment by chaadow** [comment]
LGTM 👍

**Comment by chaadow** [comment]
@Lloyd66 @ThierryISeeU Patch released ==> [v1.4.2](https://rubygems.org/gems/activestorage-openstack/versions/1.4.2)

**Pull request merged**